src/ontology/fbbt-edit.obo

```diff
@@ -146106,6 +146106,96 @@ relationship: develops_from FBbt:00059159 ! cyst cell of female germline 8-cell
+subset: cur
+is_a: FBbt:00059163 ! epithelial cell of leg disc
+relationship: expresses http://flybase.org/reports/FBgn0000157 ! Dll
+property_value: http://purl.org/dc/terms/contributor http://orcid.org/0000-0002-6095-8718
+property_value: http://purl.org/dc/terms/date 2022-09-09T12:59:07Z xsd:dateTime
+
+[Term]
+id: FBbt:00059165
+name: medial epithelial cell of leg disc
+def: "Epithelial cell of the leg disc that gives rise to the femoral and tibial segments of the leg. It is located in a ring around the central-most part of the leg disc. It expresses dachshund (Estella et al., 2012; Tse et al., 2022)." [FlyBase:FBrf0217387, FlyBase:FBrf0253840]
+subset: cur
+is_a: FBbt:00059163 ! epithelial cell of leg disc
+relationship: expresses http://flybase.org/reports/FBgn0005677 ! dac
+property_value: http://purl.org/dc/terms/contributor http://orcid.org/0000-0002-6095-8718
+property_value: http://purl.org/dc/terms/date 2022-09-09T13:07:19Z xsd:dateTime
+
+[Term]
+id: FBbt:00059166
+name: proximal epithelial cell of leg disc
+def: "Epithelial cell of the leg disc that gives rise to the coxa and the trochanter. It is located at the periphery of the leg disc. It expresses homothorax (Estella et al., 2012; Tse et al., 2022)." [FlyBase:FBrf0217387, FlyBase:FBrf0253840]
+subset: cur
+is_a: FBbt:00059163 ! epithelial cell of leg disc
+relationship: expresses http://flybase.org/reports/FBgn0001235 ! hth
+property_value: http://purl.org/dc/terms/contributor http://orcid.org/0000-0002-6095-8718
+property_value: http://purl.org/dc/terms/date 2022-09-09T13:09:48Z xsd:dateTime
+
+[Term]
+id: FBbt:00059167
+name: neuron of leg disc
+def: "Neuron that resides between the basal lamina and the disc epithelium proper of a leg disc (Beira and Paro, 2016; Tse et al., 2022)." [FlyBase:FBrf0233457, FlyBase:FBrf0253840]
+subset: cur
+intersection_of: FBbt:00001446 ! embryonic/larval neuron
+intersection_of: part_of FBbt:00001780 ! ventral thoracic disc
+property_value: http://purl.org/dc/terms/contributor http://orcid.org/0000-0002-6095-8718
+property_value: http://purl.org/dc/terms/date 2022-09-09T15:46:18Z xsd:dateTime
+
+[Term]
+id: FBbt:00059168
+name: glial cell of leg disc
+def: "Glial cell that is embedded in the epithelium of a leg disc (Tripathi and Irvine, 2022; Tse et al., 2022)." [FlyBase:FBrf0253125, FlyBase:FBrf0253840]
+subset: cur
+intersection_of: FBbt:00001251 ! embryonic/larval glial cell
+intersection_of: part_of FBbt:00001780 ! ventral thoracic disc
+property_value: http://purl.org/dc/terms/contributor http://orcid.org/0000-0002-6095-8718
+property_value: http://purl.org/dc/terms/date 2022-09-09T15:56:52Z xsd:dateTime
+
+[Term]
+id: FBbt:00059169
+name: hemocyte of leg disc
+def: "Larval sessile hemocyte that resides within a leg disc (Tse et al., 2022)." [FlyBase:FBrf0253840]
+subset: cur
+intersection_of: FBbt:00005496 ! larval sessile hemocyte
+intersection_of: part_of FBbt:00001780 ! ventral thoracic disc
```

Comment:
I would rather amend the definition of `larval sessile hemocyte`. As for the other kind of non-epithelial cells in imaginal discs (neurons/neuronal precursors, glial cells, tracheal cells, muscle precursor cells), they may not be part of the _disc epithelium proper_ (only attached to it) but are part of the larger structure of the disc.

Reply:
yeah, can often be tricky to decide whether something is an actual component of something or not, I guess one of the main considerations would be whether these cells remain here, or whether different hemocytes may come and go (like mammalian immune cells) - abstract of https://www.sciencedirect.com/science/article/pii/S0955067415000782 suggests the latter:
>Plasmatocytes, sometimes referred to with the broader term of hemocytes, migrate widely during embryonic development and cycle in the larvae between sessile and circulating positions

Reply:
It seems indeed clear that sessile hemocytes attached to the body wall, at least, are dynamic and can rejoin the circulating pool (and conversely circulating hemocytes can join the sessile compartment). However I don’t think it has been shown to also be the case for the sessile hemocytes attached to the imaginal discs — all the studies I have found on the matter focus on the body wall-associated hemocytes.

Without more information, I agree it’s safest for now to assert that the leg disc hemocytes are `attached to` the leg disc rather than `part of` it.